doc/releases/changelog-dev.md

```diff
@@ -113,6 +113,15 @@
 * Empty initialization of `PauliVSpace` is permitted.
   [(#5675)](https://github.com/PennyLaneAI/pennylane/pull/5675)
 
+* `MultiControlledX` can now be decomposed even when no `work_wires` are provided. The implementation returns $\mathcal{O}(\text{len(control\_wires)}^2)$ operations, and is applicable for any multi controlled unitary gate.
+  [(#5735)](https://github.com/PennyLaneAI/pennylane/pull/5735)
+
+* Single control unitary now includes the correct global phase.
+  [(#5735)](https://github.com/PennyLaneAI/pennylane/pull/5735)
+
+* Single control `GlobalPhase` has now a decomposition, i.e. relative phase on control wire.
```

Comment:
Is this PR restricted to *single*-controlled?

Reply:
No, but the function `ctrl_decomp_zyz` that was used but for single-controlled operations had some errors including the application of the global phase. So this PR _also_  solves that issue.

Reply:
Got it! It's not a bit problem, but I still read the changelog as if we can now only do single control GlobalPhase, rather than arbitrary control

Reply:
Oh I see what you mean now. No, you are right. Regarding the GlobalPhase operator only the single control operator  is currently implemented using the relative phase, as this is what we discussed would be necessary for the `ctrl_zyz_decomposition` (which works only for single control). 

I have just tested and the method in Lemma 7.5 that I use in `_decompose_multicontrolled_unitary` also works for multi controlled Global phase, but I am not sure is the most efficient implementation for it. As discussed previously, applying iteratively Lemma 5.2 might be a better approach.

I was thinking addressing that in a different story, what do you think?

Reply:
Resolving this conversation as we agreed to create a new story to address the multi control Global Phase decomposition